torch/_torch_docs.py

```diff
@@ -4191,6 +4242,15 @@ def merge_dicts(*dicts):
     {input}
 """.format(**common_args))
 
+add_docstr(torch.is_conj, r"""
```

Comment:
It's interesting how implementation-focused the docs are, which is not necessarily a bad thing.

From a user-facing perspective, is the bit implementing this the right focus? What if we used language like "conjugate view"? This function would then be `is_conj_view()`, for example. A lot of the language would be parallel with this approach.

Sorry if this was already discussed and I missed the party.

Reply:
"Conjugate bit" is a user facing concept, in the same way strides are a user facing concept. There isn't a better way to describe the semantic meaning here.

Reply:
OK